applepay/payment.py

```diff
@@ -62,6 +64,29 @@ def _generate_symmetric_key(self, shared_secred):
 
         return sha.digest()
 
+    @staticmethod
+    def signing_time_is_valid(signature, current_time, threshold):
+        """ Given a detached top-level CMS signature, validate the 'signingTime'
+        attribute against the current time and a time-delta threshold.
+
+        If the difference between the current time and the 'signingTime' exceeds
+        the threshold, the token should be considered invalid.
+
+        :param signature: Base64 encoded detached CMS signature data.
+        :type: str
+        :param current_time: Current system time to compare the token against.
+        :type: offset-aware datetime
+        :param threshold: Amount of time to consider the token valid.
+        :type: timedelta
+        :return: False if the signing time exceeds the threshold, otherwise True
+        :rtype: bool
+        :raises: AttributeError if no 'signingTime' attribute can be found,
+        indicating an invalid token. May also raise if signature data is in an
+        unexpected format, inconsistent with the CMS 'ContentInfo' object.
+        """
+        signing_time = payment_utils.retrieve_signature_signing_time(signature)
```

Comment:
I thought about this, but I feel that it's cleaner this way. Utils doesn't depend on data in the Payment class. Maybe moving the staticmethod to utils is better?